data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -44,74 +44,74 @@ Commands pushed onto the command queue will be executed once the JavaScript
 Tracking Client loads. After that, ``_paq.push`` becomes synchronous, meaning
 each command is executed at the moment of push.
 
-.. _jtc-api-tracker-object:
+.. _jtc-api-jstc-object:
 
-Tracker object
---------------
+JavaScript Tracking Client object
+---------------------------------
 
-Tracker object offers an alternative method of calling tracking functions.
+JavaScript Tracking Client object offers an alternative method of calling tracking functions.
 While it's more difficult to access than the :ref:`command queue<jtc-api-command-queue>`,
 it allows to read the return value of a tracking function and makes
 multi-tracker setups possible.
 
-Tracker object can be accessed using :ref:`Piwik.getTracker<jtc-api-Piwik.getTracker>`
+JavaScript Tracking Client object can be accessed using :ref:`Piwik.getTracker<jtc-api-Piwik.getTracker>`
 or :ref:`Piwik.getAsyncTracker<jtc-api-Piwik.getAsyncTracker>` function.
 
 .. _jtc-api-Piwik.getTracker:
 
 .. function:: Piwik.getTracker(trackerUrl, siteId)
 
-    Getter for Tracker object.
+    Getter for JavaScript Tracking Client object.
 
-    :param string trackerUrl: **Required** URL for Tracker
+    :param string trackerUrl: **Required** URL for JavaScript Tracking Client
     :param string siteId: **Required** Site ID that will be linked to tracked data.
-    :returns: Tracker object
+    :returns: JavaScript Tracking Client object
 
-    Example of usage (accessing Tracker object and tracking a custom event)::
+    Example of usage (accessing JavaScript Tracking Client object and tracking a custom event)::
 
-        var tracker = Piwik.getTracker("https://example.com/", "45e07cbf-c8b3-42f3-a6d6-a5a176f623ef");
-        tracker.trackEvent("video", "video-paused", "intro.mp4", 15.2);
+        var jstc = Piwik.getTracker("https://example.com/", "45e07cbf-c8b3-42f3-a6d6-a5a176f623ef");
```

Comment:
I noticed that you shorten JavaScript Tracking Client to JSTC, but wouldn't JTC make more sense?

Reply:
even if it would, `jstc` is a term from glossary and adding another aberration would not help us